src/cli/cli.hpp

```diff
@@ -748,6 +753,9 @@ class Interpreter
         {"singleton", &Interpreter::ProcessSingleton},
 #if OPENTHREAD_CONFIG_SNTP_CLIENT_ENABLE
         {"sntp", &Interpreter::ProcessSntp},
+#endif
+#if OPENTHREAD_CONFIG_SRP_SERVER_ENABLE
+        {"srpserver", &Interpreter::ProcessSrpServer},
```

Comment:
In client code I went with `srp client` as the command (`srp` as the main command and then `clinet` as sub-command and was thinking `srp server` for server). Here we have `srpserver` as a full command name.

We should converge/decide on this (make them similar). I like the `srp clinet` and `srp server` model a little bit more, but it is not a strong preference.

Thoughts? Suggestion? @wgtdkp @jwhui others?

Reply:
I was thinking we should use independent CLI commands if we have separate public headers `srp_server.h` & `srp_client.h`, but having client & server as sub-commands of `srp` also makes sense to me 😃 .

Reply:
@jwhui Do you have a preference?

Reply:
Updated the command to `srp server`.